charts/logzio-telemetry/values.yaml

```diff
@@ -944,4 +997,61 @@ opencost:
+        custom: container_oom_events_total
+      drop:
+        custom: 
+    # for applications pipeline: applications job
+    applications:
+      keep:
```

Comment:
By default, all metrics are going to be scrapped? When both `keep` and `drop` regexes are present what will overwrite?

Reply:
All metrics are scraped by default, if a user uses keep & drop, both filters will exists, and the keep filter will take precedence.